ext/sdk/resources/sdk-root/shell/src/resource-templates/ts/scaffolder.ts

```diff
@@ -0,0 +1,212 @@
+  };
+}
+
+function getTsConfigClient() {
+  return `
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "*": ["types/*"]
+    },
+    "outDir": "./",
+    "noImplicitAny": true,
+    "module": "commonjs",
+    "target": "es6",
+    "allowJs": true,
+    "lib": ["es2017"],
+    "types": ["@citizenfx/client", "@types/node"],
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "esModuleInterop": true
+  },
+  "include": ["./**/*"],
+  "exclude": ["**/node_modules"]
+}
+  `
+}
+
+
+function getTsConfigServer() {
+  return `
+  {
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "*": ["types/*"]
+    },
+    "outDir": "./",
+    "noImplicitAny": true,
+    "module": "commonjs",
+    "target": "es6",
+    "allowJs": false,
+    "lib": ["es2015", "DOM"],
+    "types": ["@citizenfx/server", "@types/node"],
+    "moduleResolution": "node",
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "resolveJsonModule": true,
+    "esModuleInterop": true
+  },
+  "include": ["./**/*"],
+  "exclude": ["**/node_modules"]
+}
+  `
+}
+
+function getWebpackContent() {
+  return `
+const webpack = require('webpack');
+const path = require('path');
+const RemovePlugin = require("remove-files-webpack-plugin");
+
+const buildPath = path.resolve(process.cwd(), 'dist');
+
+const client = {
+  entry: './client/client.ts',
+
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        use: "ts-loader",
+        exclude: /node_modules/,
+      },
+    ],
+  },
+  plugins: [
+    new webpack.DefinePlugin({ "global.GENTLY": false }),
+    new RemovePlugin({
+      before: {
+        include: [path.resolve(buildPath, "client")]
+      },
+      watch: {
+        include: [path.resolve(buildPath, "client")]
+      }
+    })
+  ],
+  optimization: {
+    minimize: true
+  },
+  resolve: {
+    extensions: [".tsx", ".ts", ".js"]
+  },
+
+  output: {
+    path: path.resolve(buildPath, "client"),
+    filename: "[contenthash].client.js"
+  }
+}
+
+const server = {
+  entry: './server/server.ts',
+
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        use: "ts-loader",
+        exclude: /node_modules/,
+      },
+    ],
+  },
+  plugins: [
+    new webpack.DefinePlugin({ "global.GENTLY": false }),
+    new RemovePlugin({
+      before: {
+        include: [path.resolve(buildPath, "server")]
+      },
+      watch: {
+        include: [path.resolve(buildPath, "server")]
+      }
+    })
+  ],
+  optimization: {
+    minimize: true
+  },
+  resolve: {
+    extensions: [".tsx", ".ts", ".js"]
+  },
+
+  output: {
+    path: path.resolve(buildPath, "server"),
+    filename: "[contenthash].server.js"
+  },
+  target: "node"
+}
+
+module.exports = [client, server];
+  `
+}
+
+function getPackageContent(resourceName: string) {
+  return `
+{
+  "name": "${resourceName}",
+  "version": "1.0.0",
+  "main": "index.js",
+  "author": "Your name",
+  "license": "MIT",
```

Comment:
Kinda weird choice of license for template, I'd expect that to be something proprietary, tho idk, @blattersturm what do you think?